### ISSUE TITLE
Temperature tag on themometer icons

### ISSUE THREAD

**Issue by daniel-mmm** [Templarian/MaterialDesign#5586]
I was looking for an icon to signify temperature but was unable to find a fitting one. I searched for temperature, cold, hot, warm etc. but couldn't find a match.

After searching in issues for requesting one I ran into some icon named thermometer something. I then searched thermometer and found what I had been looking for.

Having the thermometer icons show up when searching for temperature seems reasonable to me and would have saved me some time of searching.

**Comment by Xenomorph99** [comment]
Good catch! I just added the `temperature` alias to the `thermometer` icons. I agree, this will help people in the future if they search for "temperature".
